src/test/java/BaseTest.java

```diff
@@ -16,15 +23,12 @@ public  class BaseTest {
     public static WebDriverWait wait = null;
     public static Actions actions;
 
-   @BeforeSuite
-    public static void setupClass() {
-        WebDriverManager.chromedriver().setup();
-   }
+
 
     @BeforeMethod
     @Parameters({"baseURL"})
     public static void setUpBrowser(String baseURL) {
-        driver = new ChromeDriver();
+        driver = pickBrowser(System.getProperty("browser"));
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/215354922-ee408c8c-1777-49a7-a37e-052d261cd9a0.png)

Solution

```
    @BeforeMethod
    @Parameters({"baseURL"})
    public static void setUpBrowser(String baseURL) throws MalformedURLException {
        driver = pickBrowser(("firefox"));
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.manage().window().maximize();
        driver.get(baseURL);
        wait = new WebDriverWait(driver,Duration.ofSeconds(4));
        actions = new Actions(driver);
    }
```